tests/playwright-test/golden.spec.ts

```diff
@@ -394,6 +396,190 @@ test('should compare binary', async ({ runInlineTest }) => {
   expect(result.exitCode).toBe(0);
 });
 
+test('should throw for invalid maxDiffPixels values', async ({ runInlineTest }) => {
+  expect((await runInlineTest({
+    ...files,
+    'a.spec.js': `
+      const { test } = require('./helper');
+      test('is a test', ({}) => {
+        expect(Buffer.from([1,2,3,4])).toMatchSnapshot({
+          maxDiffPixels: -1,
+        });
+      });
+    `
+  })).exitCode).toBe(1);
+});
+
+test('should throw for invalid maxDiffPixelRatio values', async ({ runInlineTest }) => {
+  expect((await runInlineTest({
+    ...files,
+    'a.spec.js': `
+      const { test } = require('./helper');
+      test('is a test', ({}) => {
+        expect(Buffer.from([1,2,3,4])).toMatchSnapshot({
+          maxDiffPixelRatio: 12,
+        });
+      });
+    `
+  })).exitCode).toBe(1);
```

Comment:
What does the error message look like? Can you add an expect for it to make sure it remains friendly?

Reply:
added a confidence test!